Bump SonataBlockBundle to ^4.0 (#1924)
sonata-project/SonataMediaBundle: UPGRADE-4.0.md

UPGRADE FROM 3.x to 4.0
=======================

> ## Some classes and methods are now final and should not be overridden:

Reviewer: ```suggestion
## Some classes and methods were marked as final, thus they cannot be overridden:
```

> ## Some classes and methods were marked as final, thus they cannot be overridden:

Reviewer: This list only contains classes, what are the deprecated methods?

Author: Fixed.
